styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java

```diff
@@ -309,16 +305,7 @@ void awaitWorkflowInstanceCompletion(WorkflowInstance workflowInstance) {
   private void printTime() {
     LOG.info("The time is {}", now);
   }
-
-  private ScheduleSourceFactory fakeScheduleSource() {
-    return (changeListener, removeListener, environment, exec) ->
-        /* start */ () -> {
-      workflowChangeListener = changeListener;
-      workflows.forEach(changeListener);
```

Comment:
these three fields can be deleted from the class too

Reply:
Yeah. Good catch. I missed that.